optuna/storages/rdb/storage.py

```diff
@@ -450,9 +448,28 @@ def get_all_study_summaries(self) -> List[StudySummary]:
     def create_new_trial(self, study_id, template_trial=None):
         # type: (int, Optional[FrozenTrial]) -> int
 
+        return self._create_new_trial_with_trial(study_id, template_trial)[0]
+
+    def _create_new_trial_with_trial(
+        self, study_id: int, template_trial: Optional[FrozenTrial] = None
+    ) -> Tuple[int, FrozenTrial]:
+        """Create a new trial and returns trial_id and the trial converted into a FrozenTrial.
+
+        Args:
+            study_id:
+                Study id.
+            template_trial:
+                A FrozenTrial with default trial attributes.
+
+        Returns:
+            A tuple of the trial_id of the created trial and the trial converted
+            into a FrozenTrial instance.
```

Comment:
[nit] Please add an internal link to `FrozenTrial`.
```suggestion
            into a :class:`~optuna.trial.FrozenTrial` instance.
```